src/pde_solve.jl

```diff
@@ -59,5 +60,78 @@ function DiffEqBase.solve(
 
     Flux.train!(loss, ps, data, opt; cb = cb)
 
-    save_everystep ? iters : u0(X0)[1]
+
+    if give_limit == false
+        save_everystep ? iters : u0(X0)[1]
+    else
+        ## UPPER LIMIT
+        sdeProb = SDEProblem(μ , σ , X0 , prob.tspan)
+        ensembleprob = EnsembleProblem(sdeProb)
+        sim = solve(ensembleprob, EM(), EnsembleThreads(), dt=dt,trajectories=800,adaptive=false)
+        function sol_high()
+            Uo = []
+            p = nothing
+            for u in sim.u
+                xsde = u.u
+                U = g(xsde[end])
+                u = u0(x0)[1]
+                for i in length(ts)-1:-1:1
+                    t = ts[i]
+                    _σᵀ∇u = σᵀ∇u[i](xsde[i])
+                    dW = sqrt(dt)*randn(d)
+                    U = U .+ f(xsde[i], u, _σᵀ∇u, p, t)*dt .- _σᵀ∇u'*dW
+                end
+                Uo = vcat(Uo , U)
+            end
+            Uo
+        end
+        loss_() = sum(sol_high())/800
```

Comment:
why divide by 800?

Reply:
and no norm or abs on this?

Reply:
I will change 800 to user defined trajectories. In paper the sum is minimized without abs().

Reply:
Do we have to add a norm or abs?

Reply:
just took a look at the paper, yeah no abs. Interesting. So just change to number of trajectories